### PR TITLE
Updating shape sys, adding GE scale sys, and cleanup

### PR DESCRIPTION
**Pull request by gmadigan**
A number of changes, mainly: Adding GE Scale systematic variations and updating shape systematics to use new scale factors. Also cleaned up some lines, fixed some variable names, added muon and electron counts to the variations (do we want others in the _kinematicvariables list? DeepJet score, JetCount?), and added the ttv contribution missing from the pdf variations.

Questions: Do we want the diboson shape systematic to include ttV? Since they share a SF? And how does the implementation look for the GE sys? I think in the future we may want to do away with the 'MES' nomenclature and instead use just 'ROC' and 'GE.' What is labeled 'MES' right now just includes Rochester corrections, which does not include all the MES uncertainties (i.e., GE), nor is it restricted to just MES (Rochester also applies a MER smearing). Something to consider.

### CODE REVIEW

**Comment by DavidMorse** [comment]
I think TTV is small enough we don't need a shape systematic. We needed to include it in the SF to get high enough purity in the control region.

If possible I'd like to keep one single MER and MES systematic, because it is much easier to present. Let's think about how it could be implemented.....

File: LQResultProducer.py [hunk 1]
```diff
@@ -8867,11 +8901,16 @@ def ShapeSystematic(channel,normalWeight,presel,cutFile):
 				Z  = QuickIntegral(t_ZJets,selection[0]+'*'+str(Rz_uujj),1.0)
 				W  = QuickIntegral(t_WJets,selection[0],1.0)
 				tt = QuickIntegral(t_TTBar,selection[0]+'*'+str(Rtt_uujj),1.0)
-				VV = QuickIntegral(t_DiBoson,selection[0],1.0)
+				VV = QuickIntegral(t_DiBoson,selection[0]+'*'+str(Rvv_uujj),1.0)
+				TTV = QuickIntegral(t_TTV,selection[0]+'*'+str(Rvv_uujj),1.0)
+				VV += TTV
```
Comment (DavidMorse) [inline LQResultProducer.py]: QuickIntegral returns [val,err]. You have to combine the separately, using error propagation for the error

VV[0]+=TTV[0]
tmp=sqrt(VV[1]*VV[1] + TTV[1]*TTV[1])
VV[1]=tmp

File: LQResultProducer.py [hunk 2]
```diff
@@ -8867,11 +8901,16 @@ def ShapeSystematic(channel,normalWeight,presel,cutFile):
...
+
 				Z_diff  = QuickIntegral(t_ZJets,thisSel+'*'+str(Rz_diff[weight]),1.0)
 				W_diff  = W
 				tt_diff = QuickIntegral(t_TTBar,thisSel+'*'+str(Rtt_diff[weight]),1.0)
-				VV_diff = QuickIntegral(t_DiBoson,thisSel,1.0)
+				VV_diff = QuickIntegral(t_DiBoson,thisSel+'*'+str(Rvv_diff[weight]),1.0)
+				TTV_diff = QuickIntegral(t_TTV,thisSel+'*'+str(Rvv_diff[weight]),1.0)
+				VV_diff += TTV_diff
```
Comment (DavidMorse) [inline LQResultProducer.py]: same as above

**Review by DavidMorse — CHANGES_REQUESTED** [review]

File: LQResultProducer.py (hunk 1 above)
Comment (gmadigan) [reply LQResultProducer.py]: Good catch. 

"I think TTV is small enough we don't need a shape systematic. We needed to include it in the SF to get high enough purity in the control region."

Do we still want to include ttV in the VV shape systematic, in that case? Do we want to keep the VV shape systematic pure VV?

Comment (DavidMorse) [reply LQResultProducer.py]: Oh I see, you mean perform the shape systematic after you already have added TTV in? If that is the case, yes I agree go ahead and do that.

**Pull request merged**